### PR TITLE
Support for Play! 2.8

### LINKED ISSUE #298: Play 2.8 support
Soon Play 2.8 will be released and it'd be nice to have play-swagger supporting it then.

### PR DESCRIPTION
**Pull request by gabfssilva**
Fixes #298

I ran a few tests here and it worked out just fine. If there's anything I missed, let me know.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -2,8 +2,8 @@ import sbt._
 
 object Dependencies {
   object Versions {
-    val play = "2.7.3"
-    val playJson = "2.7.4"
+    val play = "2.8.0"
+    val playJson = "2.8.0"
```
Comment (thirdteeth) [inline project/Dependencies.scala]: the playJson Version should be 2.8.1

Comment (thirdteeth) [reply project/Dependencies.scala]: the sbt plugin version in src/sbt-test/sbt-play-swagger/generate-docs/project/plugins.sbt should also be upgrade to 2.8.0

**Comment by gabfssilva** [comment]
Thanks, @thirdteeth.

@kailuowang, could you take a look?

**Comment by kailuowang** [comment]
I will do the branch switch and release soon.

**Comment by kailuowang** [comment]
0.9.1-PLAY2.8 released

**Pull request merged**